app/controllers/keywords_controller.rb

```diff
@@ -1,5 +1,25 @@
 # frozen_string_literal: true
 
+require 'csv'
+
 class KeywordsController < ApplicationController
   def index; end
+
+  def create
+    keywords = load_keywords
+    Rails.logger.debug(keywords)
+    redirect_to keywords_path
+  end
+
+  private
+
+  def load_keywords
+    keywords_file = params[:keywords_file]
+    csv_data = CSV.read(keywords_file.path)
+    keywords = []
+    csv_data.each do |row|
+      keywords.append(row.first)
+    end
+    keywords
```

Comment:
How about using the `map` function? It would simplify the code a bit.
```suggestion
    csv_data.map do |row|
      row.first
    end
```

Reply:
Fixed in [`ada71a1` (#38)](https://github.com/suho/google-search-ruby/pull/38/commits/ada71a118f36016a575b128ffac04d1646c83b98)